### PR TITLE
[WIP] #3458 Change "Base Unit of Measure" to "Base Unit of Weight"

### LINKED ISSUE #3458: Change "Base Unit of Measure" to "Base Unit of Weight"
### Expected behavior
The weight UOM should be called "Base Unit of Weight" similar to how Unit of Length is called "Base Unit of Length".

### Actual behavior
It's called "Base Unit of Measure" which is probably a legacy value from when we only cared about weight.

### Steps to reproduce the behavior
1. Login as an admin
1. Observe that the UOM for weight is called "Base Unit of Measure"
1. Despair

Note: Changing this for real involves changing the unlying schema and writing migrations for that

### Versions
```
Node: 8.9.1
NPM: 5.5.1
Meteor Node: 8.9.3
Meteor NPM: 5.5.1
Reaction CLI: 0.24.2
Reaction: 1.6.3
Reaction branch: master
Docker: 17.12.0-ce
```

### PR DESCRIPTION
**Pull request by codejockie**
This PR makes some changes to shop schema and other shop related settings file. It changes "Base Unit of Measure" to "Base Unit of Weight".

Closes #3458

### CODE REVIEW

File: imports/plugins/core/versions/server/migrations/20_set_shop_uows.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import { Migrations } from "meteor/percolate:migrations";
```
Comment (pmn4) [inline imports/plugins/core/versions/server/migrations/20_set_shop_uows.js]: oooh, hadn't seen that there were migrations in meteor... this is great!

**Comment by pmn4** [comment]
lgtm

File: imports/plugins/core/versions/server/migrations/index.js [hunk 2]
```diff
@@ -16,3 +16,4 @@ import "./15_update_shipping_status_to_workflow";
 import "./16_update_billing_paymentMethod";
 import "./17_set_shop_uols";
 import "./18_use_react_for_header_and_footer_layout";
+import "./20_set_shop_uows";
```
Comment (kieckhafer) [inline imports/plugins/core/versions/server/migrations/index.js]: doesn't really matter, but where is migration 19?

Comment (zenweasel) [reply imports/plugins/core/versions/server/migrations/index.js]: I believe that was in a PR that ended up being closed/not merged

**Comment by kieckhafer** [comment]
@codejockie This looks good, is there a reason the [WIP] is still on here?

**Comment by zenweasel** [comment]
This is one of those PR's that I thought would just be a quick fix but ended being pretty impactful. Not sure if the cure isn't worse than the disease here. The naming is confusing but I doubt it's bothering anybody but me.

**Comment by codejockie** [comment]
@kieckhafer it's just as @zenweasel said. It turned out needing changes in various files than was expected.

**Comment by spencern** [comment]
@zenweasel @kieckhafer @codejockie I'd like to get this PR resolved, let's either close it as not an issue or get it ready and merge it.

**Comment by zenweasel** [comment]
Closing as fixing this bug is just not a priority right now. Also don't want changes that are going to have to be dealt with in the simple-schema PR.

**Pull request closed without merging**